blocks/api/parser.js

```diff
@@ -118,7 +119,15 @@ export function getBlockAttribute( attributeKey, attributeSchema, innerHTML, com
 			break;
 	}
 
-	return value === undefined ? attributeSchema.default : asType( value, attributeSchema.type );
+	value = applyFilters( 'blocks.getBlockAttribute.source', value, attributeSchema, innerHTML, commentAttributes );
```

Comment:
Do we really need two filters? `blocks.getBlockAttribute` and `blocks.getBlockAttribute.source`